src/config/ConfigHelper.ts

```diff
@@ -36,22 +36,12 @@ export const configHelperNetworks: Config[] = [
   },
   {
     ...configHelperNetworksBase,
-    chainId: 3,
-    network: 'ropsten',
-    nodeUri: 'https://ropsten.infura.io/v3',
-    providerUri: 'https://v4.provider.ropsten.oceanprotocol.com',
-    subgraphUri: 'https://v4.subgraph.ropsten.oceanprotocol.com',
-    explorerUri: 'https://ropsten.etherscan.io',
-    gasFeeMultiplier: 1.1
-  },
-  {
-    ...configHelperNetworksBase,
-    chainId: 4,
-    network: 'rinkeby',
```

Comment:
I removed ropsten & rinkeby support here but not sure we should. It should definitely not be used anymore but this change would break e.g. current ve testing. If we're fine with that then we can keep them out of here.

Reply:
wait. let's keep them for 2 more weeks, until 5th of oct